src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/app.module.ts

```diff
@@ -9,11 +9,11 @@ import { NavMenuComponent } from './nav-menu/nav-menu.component';
 import { HomeComponent } from './home/home.component';
 import { CounterComponent } from './counter/counter.component';
 import { FetchDataComponent } from './fetch-data/fetch-data.component';
-////#if (IndividualLocalAuth)
+//// #if (IndividualLocalAuth)
```

Comment:
I think this is an issue. I believe the templating language expects the `#` to be right after the comment mark

Reply:
So `////#if` instead of `//// #if` as it was before

Reply:
linter complains about it, so I intentionally added that space to avoid the error.
But now thinking about this this will not be in the final app, so we can just fix this.

Reply:
Yes, that's only to comply with our templating system.